integration_tests/tests/consistency/row_comparisons/consistency_daily_events.sql

```diff
@@ -5,12 +5,12 @@
 
 -- this test ensures the daily_activity end model matches the prior version
 with prod as (
-    select *
```

Comment:
Reminder to remove before merging so this is incorporated in future test cases.

Reply:
reminder to myself to remove from the other tests as well

Reply:
removed